config/software/adodbapi.rb

```diff
@@ -7,5 +7,9 @@
 
 build do
   ship_license "LGPLv2"
-  command "#{install_dir}/embedded/bin/pip install -I --install-option=\"--install-scripts=#{install_dir}/bin\" #{name}==#{version}"
-end
\ No newline at end of file
+  if ohai['platform'] == 'windows'
+    command "#{install_dir}/embedded/Scripts/pip install -I --install-option=\"--install-scripts=#{install_dir}/bin\" #{name}==#{version}"
```

Comment:
can't we use a new variable `pip_location` that we could use so we don't have this if statement for all python dependencies?